configs/wasmd/chains/ibc-1.json

```diff
@@ -1 +1,17 @@
-{"key":"testkey","chain-id":"ibc-1","rpc-addr":"http://localhost:26557","account-prefix":"wasm","gas-adjustment":1.5,"gas-prices":"0.025stake", "trusting-period":"336h"}
+{
+  "type": "cosmos",
+  "value": {
+    "key": "testkey",
+    "chain-id": "ibc-1",
+    "rpc-addr": "http://localhost:26657",
```

Comment:
Shouldnt it be 26557?

Reply:
That's the same rpc as ibc-0

Reply:
You're right. When I updated the scripts I didn't update `dev-env` to use `two-chainz-wasmd` so it ran the scripts for the cosmos tests 🤦‍♂️

Reply:
Pushing a fix for this rn. Going to just include a fix for `rly tx transfer` too since it's short.